### PR TITLE
USWDS - Address Form Template

### PR DESCRIPTION
**Pull request by chazchumley**
[Preview →](https://federalist-3b6ba08e-0df4-44c9-ac73-6fc193b0e19c.app.cloud.gov/preview/uswds/uswds/cc-address-form/components/preview/address-form.html)

## Description

Update the Address Form Template to be inline with USPS standards.
* Changed Label on Address Line 1 to "Street Number and Name"
* Added "Apartment/Floor/suite" select
* Added "Unit Number" text field
* Changed label on Zip to "Zip Code"

Issue: [Update the address template #3538 ](https://github.com/uswds/uswds/issues/3538)

### CODE REVIEW

**Review by mejiaj — APPROVED** [review]

File: src/components/07-form/templates/address-form.njk [hunk 1]
```diff
@@ -1,11 +1,19 @@
...
     <legend class="usa-legend">Mailing address</legend>
-    <label class="usa-label" for="mailing-address-1">Street address 1</label>
+    <label class="usa-label" for="mailing-address-1">Street Number and Name</label>
     <input class="usa-input" id="mailing-address-1" name="mailing-address-1" type="text">
 
-    <label class="usa-label" for="mailing-address-2">Street address 2 <span class="usa-hint">(optional)</span></label>
-    <input class="usa-input" id="mailing-address-2" name="mailing-address-2" type="text">
+    <label class="usa-label" for="apt-floor-suite">Apartment/Floor/Suite <span class="usa-hint">(optional)</span></label>
+    <select class="usa-select" id="apt-floor-suite" name="apt-floor-suite">
+        <option value>- Select One -</option>
```
Comment (thisisdano) [inline src/components/07-form/templates/address-form.njk:9]:
```suggestion
        <option value>- Select -</option>
```
Should this match the `select` [in our guidance](https://designsystem.digital.gov/components/form-controls/#dropdown)?

File: src/components/07-form/templates/address-form.njk [hunk 2]
```diff
@@ -1,11 +1,19 @@
...
+        <option value="apartment">Apartment</option>
+        <option value="floor">Floor</option>
+        <option value="suite">Suite</option>
+    </select>
+
+    <label class="usa-label" for="apt-floor-suite-no">Unit Number <span class="usa-hint">(optional)</span></label>
+    <input class="usa-input" id="apt-floor-suite-no" name="apt-floor-suite-no" type="text">
 
     <div class="grid-row grid-gap">
       <div class="mobile-lg:grid-col-8">
```
Comment (thisisdano) [inline src/components/07-form/templates/address-form.njk:19]: Should the following select be a combo box?

Comment (mejiaj) [reply src/components/07-form/templates/address-form.njk:19]: I don't think the list enough to merit a combo box.

Comment (chazchumley) [reply src/components/07-form/templates/address-form.njk:19]: I added the code to look exactly like the USPS example.  If we want it a different way then please advise.

Comment (thisisdano) [reply src/components/07-form/templates/address-form.njk:19]: Is 50+ items not enough for combo box? I'd think a state picker is ideal for combo box

**Comment by thisisdano** [comment]
Should this template include any name info? Or is that out of scope for an address template?

File: src/components/07-form/templates/address-form.njk (hunk 2 above)
Comment (mejiaj) [reply src/components/07-form/templates/address-form.njk:19]: The comment on:

> Should the following select be a combo box?

Was pointing to the Floor/Suite/Apartment select. Yeah, a combo box would work.

**Review by jAigret-Bix — APPROVED** [review]

**Comment by mejiaj** [comment]
@thisisdano @jAigret-Bix the zipcode input already has the pattern for numbers and a dash. Do we want to include `inputmode="numeric"` as well to get the numbers only on mobile devices?

**Review by thisisdano — APPROVED** [review]

**Pull request merged**